tests/conftest.py

```diff
@@ -1896,12 +1897,29 @@ async def dont_remove_caplog_handlers(request, monkeypatch):
     original_apply_config = inmanta_logging.FullLoggingConfig.apply_config
 
     def patched_apply_config(self) -> None:
```

Comment:
This method could use some documentation as to what it achieves. Do I understand correctly that it wraps the original `apply_config` to make sure we keep caplog handlers enabled? It does this by inspecting the handlers before the original `apply_config` is called so we can restore the caplog subset afterwards?